packages/modeling/src/primitives/roundedCuboid.js

```diff
@@ -4,7 +4,8 @@ const {vec2, vec3} = require('../math')
 
 const {geom3, poly3} = require('../geometry')
 
-const createCorners = (center, size, radius, segments, slice, positive) => {
```

Comment:
You need to change this back, as createCorners() is an internal function only. The center parameter is key to the generation of the polygons.

Reply:
Done, this did fix the broken test as well.